### PR TITLE
Implement Font Awesome

### PR DESCRIPTION
**Pull request by FlavioShift3**
## Changes

1. Install `nativescript-ngx-fonticon`
2. Added Font Awesome in the setup.
3. implemented in modules to use fonts

## Purpose
Facilitate use font Awesome icons. That also can help to implement other type of fonts.

##Learning
https://market.nativescript.org/plugins/nativescript-ngx-fonticon/

### CODE REVIEW

**Review by Basim-Elayan — APPROVED** [review]
LGTM. Nice Nice job,

File: src/app/app.module.ts [hunk 1]
```diff
@@ -23,6 +24,9 @@ import { ReleaseService } from '@infrastructure/services/release.service';
     ],
     imports: [
         NativeScriptModule,
+        TNSFontIconModule.forRoot({
+            'fa': require('~/fonts/font-awesome')
+        }),
```
Comment (Basim-Elayan) [inline src/app/app.module.ts:29]: I liked how is it easy to add any font.

File: src/app/features/auth/auth.module.ts [hunk 2]
```diff
@@ -10,11 +11,13 @@ import {
  * Lazy loaded feature module for all auth-related components and dependencies.
  */
 @NgModule({
+    providers: [TNSFontIconService],
     declarations: [
         authComponents
     ],
     imports: [
         AuthRoutingModule,
+        TNSFontIconModule,
```
Comment (Basim-Elayan) [inline src/app/features/auth/auth.module.ts:20]: Might we need to move this to SharedModule later

Comment (FlavioShift3) [reply src/app/features/auth/auth.module.ts:20]: We have to have this in all modules.

Comment (Basim-Elayan) [reply src/app/features/auth/auth.module.ts:20]: Right, But it would be nice to have it in SharedModule later.

Comment (FlavioShift3) [reply src/app/features/auth/auth.module.ts:20]: Agree

**Review by Basim-Elayan — APPROVED** [review]
Thanks for adding that section in read me file.

**Pull request merged**